liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/StartDebugMojoSupport.java

```diff
@@ -289,10 +308,13 @@ protected void runLibertyMojoDeploy(boolean forceLooseApp) throws MojoExecutionE
         if(forceLooseApp) {
             Xpp3Dom looseApp = config.getChild("looseApplication");
             if (looseApp != null && "false".equals(looseApp.getValue())) {
-                log.warn("Overriding liberty plugin pararmeter, \"looseApplication\" to \"true\" and deploying application in looseApplication format");
+                log.warn("Overriding liberty plugin parameter, \"looseApplication\" to \"true\" and deploying application in looseApplication format");
                 looseApp.setValue("true");
             }
         }
+        if (LooseWarApplication.isExploded(project)) {
```

Comment:
Hmm.. does this answer my question about why you didn't have this in DevMojo ?

Reply:
Yes 👍